cumulusci/tasks/salesforce/tests/test_EnsureRecordTypes.py

```diff
@@ -253,6 +253,33 @@ def test_no_action_if_existing_record_types(self):
         self.assertFalse(task.options["generate_business_process"])
         self.assertFalse(task.options["generate_record_type"])
 
+    def test_second_rt_if_force_create(self):
+        # Asserts a second Record Type is added even when RTs already exist when force_create is True
+        task = create_task(
+            EnsureRecordTypes,
+            {
+                "record_type_developer_name": "NPSP_Default",
+                "record_type_label": "NPSP Default",
+                "record_type_description": "Default Account Record Type created by NPSP.",
+                "sobject": "Account",
+                "force_create": True,
+            },
+        )
+
+        task.sf = mock.Mock()
+        task.sf.Account = mock.Mock()
+        task.sf.Account.describe = mock.Mock(return_value=OPPORTUNITY_DESCRIBE_WITH_RTS)
```

Comment:
I'd love a comment here to call out that there's no impact from using the "wrong" describe (all it's doing is representing Record Types as present).